legal-api/src/legal_api/models/filing.py

```diff
@@ -1011,7 +1011,7 @@ def get_completed_filings_for_colin(limit=20, offset=0):
                 Filing._status == Filing.Status.COMPLETED.value,
                 Filing._source == Filing.Source.LEAR.value,
                 Filing.effective_date != None   # pylint: disable=singleton-comparison # noqa: E711;
-            ).order_by(Filing.effective_date, Filing.id).limit(limit).offset(offset).all()
+            ).order_by(Filing.transaction_id).limit(limit).offset(offset).all()
```

Comment:
seems like effective date can be in past for (at least for COD), so cannot use that for priority.